traffic_ops/v4-client/deliveryservice.go

```diff
@@ -155,6 +155,19 @@ func (to *Session) GetDeliveryServicesV30WithHdr(header http.Header, params url.
 	return data.Response, reqInf, err
 }
 
+// GetDeliveryServicesV40WithHdr returns all (tenant-visible) Delivery Services that
+// satisfy the passed query string parameters. See the API documentation for
+// information on the available parameters.
+func (to *Session) GetDeliveryServicesV40WithHdr(header http.Header, params url.Values) ([]tc.DeliveryServiceV40, ReqInf, error) {
```

Comment:
These methods should also use the `DeliveryServiceV4` alias as input/output. We should also consider implementing the new proposed TO Go client best practices (or make a mental note to come back here and change them once the groundwork is laid out for that).